### PR TITLE
feat: LLM more swap analytics 

### PR DESCRIPTION
**Pull request by cjordan-ledger**
### 📝 Description

Adds additional could/should event tags as documented [here](https://ledgerhq.atlassian.net/wiki/spaces/PTX/pages/edit-v2/3931340809)

### ❓ Context

- **Impacted projects**:  LLM
- **Linked resource(s)**:  https://ledgerhq.atlassian.net/browse/LIVE-4990

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/screens/Swap/Form/Modal/index.tsx [hunk 1]
```diff
@@ -55,6 +58,12 @@ export function Modal({
 
   const onError = useCallback(
     ({ error, swapId }) => {
+      track("error_message", {
+        ...sharedSwapTracking,
+        message: "drawer_error",
+        page: "Page Swap Drawer",
+        error: error?.name ?? "unknown",
```
Comment (sarneijim) [inline apps/ledger-live-mobile/src/screens/Swap/Form/Modal/index.tsx:65]: any specific case, why are we using nullish coalescing operator ? (only curiosity)

File: apps/ledger-live-mobile/src/screens/Swap/Form/index.tsx [hunk 2]
```diff
@@ -46,13 +46,13 @@ import {
...
 } from "../../../actions/swap";
-import { TrackScreen, track, useAnalytics } from "../../../analytics";
+import { TrackScreen, useAnalytics } from "../../../analytics";
 import { Loading } from "../Loading";
 import { NotAvailable } from "./NotAvailable";
 import { TxForm } from "./TxForm";
 import { Summary } from "./Summary";
 import { Requirement } from "./Requirement";
-import { trackSwapError, SWAP_VERSION } from "../utils";
+import { SWAP_VERSION, sharedSwapTracking, useTrackSwapError } from "../utils";
```
Comment (sarneijim) [inline apps/ledger-live-mobile/src/screens/Swap/Form/index.tsx:55]: Can we remove SWAP_VERSION and use sharedSwapTracking across all this file?

**Review by sarneijim — COMMENTED** [review]
Here, we are tracking only BTC and ETH families. I know it is not in the "track" description explicitly but we need to track any kind of family.
(I do not think Amine knows all the "small families variations", we can track those events directly using the same interactions that we are doing for the rest of the app).
Do we want add the track here or in another pr? (In case we want to add them in another pr we need update track folder to do not forget it).

File: apps/ledger-live-mobile/src/screens/Swap/Form/index.tsx (hunk 2 above)
Comment (cjordan-ledger) [reply apps/ledger-live-mobile/src/screens/Swap/Form/index.tsx:55]: We still need it as it's passed to the `TrackScreen` component as a prop

Comment (sarneijim) [reply apps/ledger-live-mobile/src/screens/Swap/Form/index.tsx:55]: Can we use "...sharedSwapTracking"?

Comment (cjordan-ledger) [reply apps/ledger-live-mobile/src/screens/Swap/Form/index.tsx]: https://github.com/LedgerHQ/ledger-live/pull/2275/commits/edb0317898063389a1913690ac7923d88944c5a5

**Review by sarneijim — APPROVED** [review]

**Pull request merged**